Manual.md

```diff
@@ -1272,6 +1295,329 @@ cache during the install/removal of the package
+as a directory, all directories under that directory have their absolute path
+passed to the trigger.
+
+<a id="triggers_gtk_immodules"></a>
+#### gtk-immodules
+
+The gtk-immodules trigger is responsible for updating the IM (Input Method) modules
+file for gtk+.
+
+During installation it uses `gtk-query-immodules-2.0 --update-cache` to update the
+cache file. It also removes the obsolete configuration file  `etc/gtk-2.0/gtk.immodules`
+if present.
+
+During removal of the `gtk+` package it removes the cache file which is located at
+`usr/lib/gtk-2.0/2.10.0/immodules.cache`.
+
+It is automatically added to packages that have `/usr/lib/gtk-2.0/2.10.0/immodules`
+present as a directory.
+
+<a id="triggers_gtk_pixbuf_loaders"></a>
+#### gtk-pixbuf-loaders
+
+gtk-pixbuf-loaders is the old name for the current `gdk-pixbuf-loaders` trigger and is
+in the process of being removed. It currently re-execs into `gdk-pixbuf-loaders` as a
+compatibility measure.
+
+For information about how it works refer to [gdk-pixbuf-loaders](#triggers_gdk_pixbuf_loaders).
+
+<a id="triggers_gtk3_immodules"></a>
+#### gtk3-immodules
+
+The gtk3-immodules trigger is responsible for updating the IM (Input Method) modules
+file for gtk+3.
+
+During installation it executes `gtk-query-immodules-3.0 --update-cache` to update the
+cache file. It also removes the obsolete configuration file  `etc/gtk-3.0/gtk.immodules`
+if present.
+
+During removal of the `gtk+3` package it removes the cache file which is located at
+`usr/lib/gtk-3.0/3.0.0/immodules.cache`.
+
+It is automatically added to packages that have `/usr/lib/gtk-3.0/3.0.0/immodules`
+present as a directory.
+
+<a id="triggers_hwdb.d_dir"></a>
+#### hwdb.d-dir
+
+The hwdb.d-dir trigger is responsible for updating the hardware database.
+
+During installation and removal it runs `usr/bin/udevadm hwdb --root=. --update`.
+
+It is automatically added to packages that have `/usr/lib/udev/hwdb.d` present
+as a directory.
+
+<a id="triggers_info_files"></a>
+#### info-files
+
+The info-files trigger is responsible for registering and unregistering the GNU info
+files for a package.
+
+It checks the existence of the info files presented to it and if it is running under
+another architecture.
+
+During installation it uses `install-info` to register info files into
+`usr/share/info`.
+
+During removal it uses `install-info --delete` to remove the info files from the
+registry located at `usr/share/info`.
+
+If it is running under another architecture it tries to use the host `install-info`
+utility.
+
+<a id="triggers_kernel_hooks"></a>
+#### kernel-hooks
+
+The kernel-hooks trigger is responsible for running whatever executable files are
+found for it's target.
+
+The targets available are pre-install, pre-remove, post-install and post-remove.
+
+When run it will try to run all executables found under `etc/kernel.d/$TARGET`. The
+`TARGET` variable is one of the 4 targets available for the trigger. It will also
```

Comment:
Dunno if that goes to deep into the triggers' logic (or if people should just read the triggers' source), but maybe it'd make sense to also list the targets here?

Reply:
the targets are listed in the paragraph above

Reply:
Oh, oops